app/models/candidate.rb

```diff
@@ -2,7 +2,9 @@ class Candidate < ApplicationRecord
   # Only Devise's :timeoutable module is enabled to handle session expiry
   # Custom Warden strategy is used instead see app/warden/magic_link_token.rb
   devise :timeoutable
-  validates :email_address, presence: true, uniqueness: true, length: { maximum: 250 }
+
+  before_validation :downcase_email
+  validates :email_address, presence: true, uniqueness: { case_sensitive: false }, length: { maximum: 250 }
```

Comment:
This makes Rails use LOWER() when comparing in the database. For handling existing email addresses in the database that are not lowercase. In theory we could allow any case, as long as the comparison is case-insensitive, but I think it's neater to be consistent.